contrib/lang/sml/packages.el

```diff
@@ -0,0 +1,44 @@
+;;; packages.el --- sml Layer packages File for Spacemacs
+;;
+;; Copyright (c) 2012-2014 Sylvain Benner
+;; Copyright (c) 2014-2015 Keith Simmons & Contributors
+;;
+;; Author: Keith Simmons <keith@the-simmons.net>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(setq sml-packages
```

Comment:
When I tested this locally using the latest version of Spacemacs, I got a "type error" referring to the docstring at the end of this definition. Removing the `"List of all...."` argument fixed it.